package/network/config/netifd/files/usr/libexec/network/packet-steering.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/sh
+
 NPROCS="$(grep -c "^processor.*:" /proc/cpuinfo)"
 [ "$NPROCS" -gt 1 ] || exit
 
-PROC_MASK="$(( (1 << $NPROCS) - 1 ))"
+PROC_MASK="$(( (1 << NPROCS) - 1 ))"
 
 find_irq_cpu() {
-	local dev="$1"
-	local match="$(grep -m 1 "$dev\$" /proc/interrupts)"
-	local cpu=0
+        dev="$1"
+	    match="$(grep -m 1 "$dev\$" /proc/interrupts)"
+	    cpu=0
 
 	[ -n "$match" ] && {
-		set -- $match
+            set -- "$match"
 		shift
-		for cur in $(seq 1 $NPROCS); do
+		for cur in $(seq 1 NPROCS); do
```

Comment:
This is not arithmetic, the $ is required here.

Reply:
fixed